terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -938,6 +938,20 @@ def qq(*words):
                             f'arn:aws:kms:{aws.region_name}:{aws.account}:key/*',
                             f'arn:aws:kms:{aws.region_name}:{aws.account}:alias/*'
                         ]
+                    },
+
+                    # SSM agent requires explicit permissions for bucket access
+                    # if S3 traffic is routed through a VPC gateway endpoint.
```

Comment:
```suggestion
                    # The SSM agent requires explicit permission to access certain AWS-managed buckets
                    # if S3 traffic is routed through a VPC gateway endpoint.
```

Add link to docs if possible.